build.sbt

```diff
@@ -25,7 +25,7 @@ val exhibitorOptionalDependencies = Seq(
   "com.typesafe.akka" %% "akka-http-spray-json" % akkaHttpVersion
 ).map(_ % Provided)
 
-val curatorVersion = "2.11.0"
+val curatorVersion = "2.12.0"
```

Comment:
Great job @rajcspsg! Is there any reason this isn't updated to `4.0.0`?

Reply:
@briantopping - I noticed the latest version of curator 4.x.x has released but it has dependency of zookeeper-3.5.1-alpha release.
I don't want to include alpha release and  our project still uses zookeeper 3.4.x
I thought we should update the dependencies incrementally so that people will have choice to pick the version of this library for their dependencies project and it won't give too much evictions.

Reply:
Excellent point, good eye. In my case, I was looking solely at a major version difference in a transitive dependency that was repaired. I'm also not going straight to production, so a CNFE from API mismatch is more likely than finding an actual bug. 

I'll close this review and watch for changes that would indicate the 4.x series is more stable. Thanks again!